runelite-client/src/main/java/net/runelite/client/ui/overlay/infobox/Timer.java

```diff
@@ -96,4 +96,9 @@ public boolean cull()
 		return timeLeft.isZero() || timeLeft.isNegative();
 	}
 
+	public long secondsPassed()
```

Comment:
I dont think this method is necessary, you can just do that outside from method in plugin

Reply:
Agreed, don't want to change Timer.java. I am having trouble extracting time left without adding another function to timer. I still need to somehow get `Duration.between(Instant.now(), endTime).getSeconds()` from the Timer. I could call the get text and convert the value back into seconds?

Reply:
Yeah, getter for endTime seems fine